src/Ledger/Introduction.lagda

```diff
@@ -53,10 +56,30 @@ private variable
 \begin{code}
 record Computational (_⊢_⇀⦇_,X⦈_ : C → S → Sig → S → Set) : Set where
   field
-    compute     : C → S → Sig → Maybe S
```

Comment:
I think it's better to leave the presentation in the PDF in the old style because it should be easier to understand by people who don't know too much type theory.

I do however like adding a few of the theorems like you did here. Maybe we could prove them by translating this version of `Computational` into the new one?

Reply:
Ok, I'll revert this back to the old Computational and add the `nothing⇒∀¬STS` property.

Reply:
Thank you!